spec/rubocop/cop/style/multiline_list_spec.rb

```diff
@@ -0,0 +1,419 @@
+# encoding: utf-8
+
+require 'spec_helper'
+
+describe RuboCop::Cop::Style::MultilineList, :config do
+  subject(:cop) { described_class.new(config) }
+
+  context 'method parameters' do
+    let(:cop_config) do
+      {
+        'EnableMethodParams' => true
+      }
+    end
+
+    context 'params listed on the first line' do
+      let(:source) do
+        ['def foo(bar,',
+         '  baz)',
+         '  do_something',
+         'end']
+      end
+
+      it 'detects the offense' do
+        inspect_source(cop, source)
+
+        expect(cop.offenses.length).to eq(1)
+        expect(cop.offenses.first.line).to eq(1)
```

Comment:
You can add `expect(cop.highlights).to eq(['bar'])` at least in some places where offenses are detected, so we see that the right piece of code is highlighted in the reporting.